test/askContainer/commands/listSkills.test.ts

```diff
@@ -0,0 +1,104 @@
+import * as vscode from "vscode";
+import * as assert from "assert";
+import * as sinon from "sinon";
+import * as model from "ask-smapi-model";
+import { CustomSmapiClientBuilder } from "ask-smapi-sdk";
+
+import { ListSkillsCommand } from "../../../src/askContainer/commands/listSkills";
+import { TelemetryClient } from "../../../src/runtime/lib/telemetry";
+import { SmapiClientFactory, SmapiResource } from "../../../src/runtime";
+import * as profileHelper from "../../../src/runtime/lib/utils/profileHelper";
+
+describe("Command_listSkills tests", () => {
```

Comment:
nit: Better to add the whole command name. For eg : `ask.container.listSkills`

Reply:
Good point, will apply to other tests.